### PR TITLE
Add report page

### PR DESCRIPTION
**Pull request by domlimm**
LET'S GO DREAM TEAM

### CODE REVIEW

File: app/report/page.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      }
+    ],
+    locale: 'en-US',
+    type: 'website'
+  }
+};
+
+const Report = () => {
+  return (
+    <Flex w='100%' h='100vh' flex={1} direction='column'>
```
Comment (marcuspeh) [inline app/report/page.tsx]: Why are u setting the height to 100vh? This page is meant to be one page and needs to be scrollable.

File: components/Report/Cards.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  data: ResultInfo;
+}
+
+const Card: React.FC<CardProps> = props => {
+  const { title, resultTab, tipsTab, result } = props.data;
+
+  return (
+    <Accordion allowToggle boxShadow={'xl'} borderRadius={'4'}>
+      <AccordionItem>
+        <h1>
```
Comment (marcuspeh) [inline components/Report/Cards.tsx]: Why are u wrapping AccordionButton in h1?

File: components/Report/Summary.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const Summary = () => {
+  return (
+    <Box bg='#EAEBFF' padding='4' borderRadius={'4'}>
+      <Text as='b' fontSize={'24'}>
+        {'Summary: {Lipid Panel}'}
+      </Text>
+      <Text fontSize={'16'}>
+        Panel of blood tests used to screen for{' '}
+        <Text as='b'>cardiovascular diseases</Text>.
```
Comment (marcuspeh) [inline components/Report/Summary.tsx]: Can you just use `<b>cardiovascular diseases</b>` to bold it?

File: components/Report/index.tsx [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export { default as Cards } from './Cards';
+export { default as RangeSection } from './RangeSection';
+export { default as Summary } from './Summary';
```
Comment (marcuspeh) [inline components/Report/index.tsx:3]: Don't need use default. Can use `export Summary from './Summary';`

**Review by marcuspeh — COMMENTED** [review]
Some changes to be done

File: components/Report/Cards.tsx (hunk 2 above)
Comment (domlimm) [reply components/Report/Cards.tsx]: Good point. It doesn't make a difference. Came from the docs.

File: app/report/page.tsx (hunk 1 above)
Comment (domlimm) [reply app/report/page.tsx]: Thanks for catching this!

File: components/Report/index.tsx (hunk 4 above)
Comment (domlimm) [reply components/Report/index.tsx:3]: Don't think that works, or am I missing something

Comment (marcuspeh) [reply components/Report/index.tsx:3]: let merge in and see

**Review by marcuspeh — APPROVED** [review]
LGTM

**Pull request merged**